src/multikey.rs

```diff
@@ -160,7 +160,7 @@ impl fmt::Display for DecodeLegacyError {
 impl std::error::Error for DecodeLegacyError {}
 
 /// The secret counterpart to Multikey
-#[derive(Debug, PartialEq, Eq, Clone)]
+#[derive(Debug, Clone)]
 pub struct Multisecret(SecretKey);
```

Comment:
I think this should be `Multisecret(Keypair)`. The libsodium (and thus ssb) "secret key" is actually the public+secret keys, concatenated. The ssb-crypto `SecretKey` type is only half of what you want.

Reply:
:open_mouth: 
Nice